addons/account_bank_statement_import/account_journal.py

```diff
@@ -6,7 +6,7 @@
 class AccountJournal(models.Model):
     _inherit = "account.journal"
 
-    bank_statements_source = fields.Selection(selection_add=[("file_import", "File Import")])
+    bank_statements_source = fields.Selection(selection_add=[("file_import", "Import (CSV, QIF, CAMT, OFX)")])
```

Comment:
should still be dynamic based on installed modules... 

you can use a function that'll be inherited in all modules